pages/apim/3.x/kubernetes/apim-kubernetes-operator-user-guide-api-context.adoc

```diff
@@ -0,0 +1,202 @@
+[[apim-kubernetes-operator-user-guide-api-context]]
+= How to use the Api Context custom resource
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_user_guide_api_context.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+[label label-version]#BETA release#
+
+== How to use the Api Context (`ApiContext`) custom resource
+
+To enable synchronization of CRDs with a remote link:https://docs.gravitee.io/apim/3.x/apim_overview_architecture.html[management API^], you need to create an Api Context referring to an existing link:https://docs.gravitee.io/am/current/am_adminguide_organizations_and_environments.html[organization and environment^].
+
+You can create multiple Api Contexts, each targeting a specific environment and defined in a specific organization of a management API instance.
+
+An Api Context is composed of two sub objects, the management and the values objects.
+
+=== The Management object
+The management object can authenticate to your management API instance by using either basic authentication or a bearer token. Authentication credentials may either be added inline in the Management Context definition or referenced from a Kubernetes secret.
+
+NOTE: If both credentials and a bearer token are defined in your custom resource, the bearer token will take precedence.
+
+=== The Values object
+The values object is used to define a key value map. Those values are used to make your Kubernetes manifest reusable and dynamic. You can use link:https://pkg.go.dev/text/template?utm_source=godoc[Go templating] to build your Api Definition manifests.
+
+== Examples
+
+The custom resource in the example below refers to a Management API instance exposed at `+https://gravitee-api.acme.com+`, and targets the `dev` environment of the `acme` organization, with the `admin` account, using basic authentication credentials defined in a Kubernetes secret:
```

Comment:
```suggestion
The custom resource in the example below refers to a Management API instance exposed at `+https://gravitee-api.example.com+`, and targets the `dev` environment of the `example-org` organization, with the `admin` account, using basic authentication credentials defined in a Kubernetes secret:
```
`example.com` should be used for examples in documentation. See  [RFC2606](https://www.rfc-editor.org/rfc/rfc2606.html). `acme.com` should be replaced with `example.com` throughout, and `acme` should be replaced with `example-org` throughout.

Reply:
All acme has been replaced 👍🏼